app/views/email_alert_subscriptions/new.html.erb

```diff
@@ -9,38 +10,43 @@
 
 <div class="govuk-width-container">
   <%= form_tag email_alert_subscriptions_path(subscriber_list_params: subscriber_list_params), class: 'signup-choices' do %>
-    <%= render partial: 'govuk_publishing_components/components/title', locals: {
-      title: @signup_presenter.name,
-      context: "Email alert subscription"
-    } %>
-
     <% if @signup_presenter.can_modify_choices? %>
+      <%= render partial: 'govuk_publishing_components/components/title', locals: {
+        title: @signup_presenter.name,
+      } %>
+
       <%= render "govuk_publishing_components/components/checkboxes", {
         name: nil,
-        heading: @signup_presenter.name,
-        hint_text: "Choose the email alerts you need.",
+        no_hint_text: true,
+        heading: t("email_alert_subscriptions.new.question"),
         items: @signup_presenter.choices_formatted,
-        visually_hide_heading: true
       } %>
-    <% end %>
-    <% if @error_message.present? %>
-      <p class="signup-choices__message"><%= @error_message %></p>
+
+      <% if @error_message.present? %>
+        <p class="signup-choices__message"><%= @error_message %></p>
+      <% end %>
+
+    <% else %>
+
+      <%= render partial: 'govuk_publishing_components/components/title', locals: {
+        title: t("email_alert_subscriptions.new.title"),
+      } %>
+
+      <p class="govuk-body">
+        <%= t("email_alert_subscriptions.new.description") %>
+      </p>
+
+      <p class="govuk-body govuk-!-margin-bottom-8">
+        <strong style="font-weight: bold"><%= @signup_presenter.name %></strong>
```

Comment:
Oh dear - is there an overly zealous CSS reset? 

There seems to be a govuk-frontend override that can be used for it (not sure if we expose it or not) of `govuk-!-font-weight-bold` which might look a bit more informed than an inline style.

Whatever decided, it'd be good to include some information about it in the commit message.

Reply:
Oh yes, thanks for spotting. It was irritating to find `<strong>` isn't so in this app. Your suggestion does work and I agree it's a better indicator of intent, so I've used that instead, and fixed up the commit message (with links).

Reply:
There is an overide class in the [design system](https://design-system.service.gov.uk/styles/typography/#font-weight) that you can apply to the `<p>` to make text bold. 

It's done like this:
```
<p class="govuk-body govuk-!-font-weight-bold">govuk-!-font-weight-bold</p>
```

Reply:
I agree we could just use a `<p>` element. However, I think `<strong>` is semantically clearer, and should be what we use for important content - it's just a shame our apps are working against us here.

Reply:
Okay yea, but, can we apply the `govuk-!-font-weight-bold` override class to the `<strong>` element so we're consistent and get any changes in the future if we change the appearance of bold on GOV.UK.

Reply:
Oh yes, I already did that. Sorry, it's not very clear in the GitHub UI - you can see the top of the panel here has an "outdated" flag on it, which means you'll need to check in "Files changed" to see the update.

Reply:
Perfect! GitHub's UI 🙄